src/lib/services/groups/service/groups.ts

```diff
@@ -1,7 +1,6 @@
 import type { operations, components } from '@globus/types/groups';
 import { ID, SCOPES } from '../config.js';
-import { build } from '../../../core/url.js';
-import { fetchWithScope } from '../../../core/fetch.js';
+import { serviceRequest } from '../../shared.js';
```

Comment:
There's several references to `shared.js` but the filename is now `shared.ts`. Sorta surprised there's no error from this so I decided to investigate and I think we have some issues with module resolution in our tsconfig. I'm looking into it now, might be nothing though because all of this works :shrug:

Reply:
From what I understand, this is the only way for Typescript to emit valid ESM (requires extensions). If you leave off the extension (or even use `.ts`), the bundle will be incorrect.

Some more details here: https://github.com/microsoft/TypeScript/issues/42151

Reply:
Wow that thread and the connected issues make for some wild reading. I guess I understand the maintainers points about not messing with JS standards but I also find myself agreeing with the people saying that writing "incorrect" Typescript code is wrong. Regardless thanks for the reference